AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1612,15 +1611,6 @@ - (void)animateLayoutTransition:(id<ASContextTransitioning>)context
       CGRect fromFrame = [context initialFrameForNode:subnode];
       CGRect toFrame = [context finalFrameForNode:subnode];
       if (CGSizeEqualToSize(fromFrame.size, toFrame.size) == NO) {
-        // To crossfade resized subnodes, show a snapshot of it on top.
-        // The node itself can then be treated as a newly-inserted one.
-        UIView *snapshotView = [subnode.view snapshotViewAfterScreenUpdates:YES];
-        snapshotView.frame = [context initialFrameForNode:subnode];
-        snapshotView.alpha = 1;
-        
-        [node.view insertSubview:snapshotView aboveSubview:subnode.view];
-        [removedViews addObject:snapshotView];
-        
         [insertedSubnodes addObject:subnode];
```

Comment:
@appleguy You are totally right!! I would like to add enable and disabling `groupOpacity` tonight / tomorrow morning into this PR and than let's land it.

Let's address the follow up steps in another PR.